client/core/core.go

```diff
@@ -3671,6 +3671,9 @@ func handleRevokeOrderMsg(c *Core, dc *dexConnection, msg *msgjson.Message) erro
 
 	tracker.revoke()
 
+	details := fmt.Sprintf("Order %v revoked.", oid)
```

Comment:
Something like...

```
fmt.Sprintf("Order %s on market %s at %s has been revoked by the server", tracker.token, tracker.mktID, dc.acct.host)
```

Reply:
For future work we'll need to add details and reaons.  This will require passing more data around in a number of places e.g. swapper->unbookHook